zeroeq/http/server.cpp

```diff
@@ -78,6 +80,7 @@ std::string _convertEndpointName(const std::string& endpoint)
 const std::string JSON_TYPE = "application/json";
 const std::string REQUEST_REGISTRY = "registry";
 const std::string REQUEST_SCHEMA = "schema";
+const std::string HTTP_SERVER_SERVICE = "_http._tcp";
```

Comment:
Also, add some documentation in the header of the class that this features exists.